collectors/bzimport/convertors.py

```diff
@@ -784,12 +817,35 @@ def get_affects(self, flaw):
             self.record_errors(errors)
             affects.append(affect_obj)
 
+            # AffectCVSS is created here because is requires an affect object
+            for cvss_pair in [
+                ("cvss2", AffectCVSS.CVSSVersion.VERSION2),
+                ("cvss3", AffectCVSS.CVSSVersion.VERSION3),
+            ]:
+                cvss = cvss_pair[0]
+                version = cvss_pair[1]
```

Comment:
nitpick: you can shorten this (and avoid indexing) by `cvss, version = cvss_pair`